pom.xml

```diff
@@ -479,6 +479,13 @@ flexible messaging model and an intuitive client API.</description>
         <artifactId>aspectjweaver</artifactId>
         <version>${aspectj.version}</version>
       </dependency>
+      
+	  <!-- TODO: should we callout BSD license: https://github.com/electronicarts/ea-agent-loader -->
```

Comment:
@merlimat Do you think we should add BSD license into [NOTICE](https://github.com/apache/incubator-pulsar/blob/master/NOTICE) file?